guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc

```diff
@@ -0,0 +1,386 @@
+    --shape-name MySQL.VM.Standard.E3.1.8GB \
+    --configuration-id ocid1.mysqlconfiguration.oc1..aaaaaaaalwzc2a22xqm5... \
+    --subnet-id ocid1.subnet.oc1.iad.aaaaaaaaxgumfzpn6sul... \
+    --admin-username <your username> \
+    --admin-password <your password> \
+    --availability-domain nFuS:US-ASHBURN-AD-1 \
+    --data-storage-size-in-gbs 50 \
+    --display-name Micronaut_Guide_MySQL
+----
+
+The response should look like this:
+
+[source,json]
+----
+{
+  "data": {
+    "analytics-cluster": null,
+    "availability-domain": "nFuS:US-ASHBURN-AD-1",
+    "backup-policy": {
+      "defined-tags": null,
+      "freeform-tags": null,
+      "is-enabled": true,
+      "retention-in-days": 7,
+      "window-start-time": "07:11"
+    },
+    "channels": [],
+    "compartment-id": "ocid1.compartment.oc1..aaaaaaaarkh3s2wcxbbm...",
+    "configuration-id": "ocid1.mysqlconfiguration.oc1..aaaaaaaalwzc2a22xqm5...",
+    "current-placement": {
+      "availability-domain": null,
+      "fault-domain": null
+    },
+    "data-storage-size-in-gbs": 50,
+    "defined-tags": {},
+    "description": null,
+    "display-name": "Micronaut_Guide_MySQL",
+    "endpoints": [],
+    "fault-domain": null,
+    "freeform-tags": {},
+    "heat-wave-cluster": null,
+    "hostname-label": null,
+    "id": "ocid1.mysqldbsystem.oc1.iad.aaaaaaaa2pq3a37hftut...",
+    "ip-address": null,
+    "is-analytics-cluster-attached": false,
+    "is-heat-wave-cluster-attached": false,
+    "is-highly-available": false,
+    "lifecycle-details": null,
+    "lifecycle-state": "CREATING",
+    "maintenance": {
+      "window-start-time": "WEDNESDAY 07:09"
+    },
+    "mysql-version": null,
+    "port": null,
+    "port-x": null,
+    "shape-name": "MySQL.VM.Standard.E3.1.8GB",
+    "source": null,
+    "subnet-id": "ocid1.subnet.oc1.iad.aaaaaaaaxgumfzpn6sul...",
+    "time-created": "2022-02-03T16:39:02.762000+00:00",
+    "time-updated": "2022-02-03T16:39:02.762000+00:00"
+  },
+  "etag": "0dea57803672c12742f7710f342bf...",
+  "opc-work-request-id": "ocid1.mysqlworkrequest.oc1.iad.eb66e373-7274-40a..."
+}
+----
+
+Note that the value for `lifecycle-state` is `CREATING`, so not all information (e.g. IP address) is available yet.
+
+Save the database OCID from the `id` property in the response.
+
+=== Retrieve Database Info
+
+It will take a few minutes for the database to finish provisioning. Wait a bit and run the https://docs.oracle.com/en-us/iaas/tools/oci-cli/2.10.4/oci_cli_docs/cmdref/mysql/db-system/get.html[get] command with the database OCID:
+
+[source,bash]
+----
+oci mysql db-system get \
+    --db-system-id ocid1.mysqldbsystem.oc1.iad.aaaaaaaa2pq3a37hftut...
+----
+
+Once the `lifecycle-state` is `ACTIVE` you can make note of the IP address and port (it should be the default value of 3306):
+
+[source,json]
+----
+"ip-address": "10.0.0.6",
+"lifecycle-state": "ACTIVE",
+"port": 3306,
+"port-x": 33060
+----
+
+external:micronaut-cloud-database-base/create-app.adoc[]
```

Comment:
I don't understand the benefit of using this instead of a common snippet. Why did we needed `external`. Could not we just have?

```
common:micronaut-cloud-database-create-app.adoc[]
```

Reply:
Resolved via https://github.com/micronaut-projects/micronaut-guides/pull/743